src/apps/vpn/ui/screens/getHelp/developerMenu/ViewDeveloperMenu.qml

```diff
@@ -290,6 +290,15 @@ MZViewBase {
             }
         }
 
+        MZButton {
+            id: resetTipsAndTricksModal
+
+            text: "Reset Intro Tips and Tricks Modal"
+            onClicked: {
+                MZSettings.tipsAndTricksIntroShown = false
+            }
+        }
```

Comment:
This also feels unnecessary given the whole tips and tricks page is planned to be removed soon.

Reply:
Do you think there is a harm to including this until we remove tips and tricks? Without this, the only way to test this is to uninstall the app completely, it seemed helpful for testing to have a way to do this without having to log back in. I recognize this code may be pulled in the next few months, but still seems helpful for now.